docs/howtos/recipes_xbk_inc.md

```diff
@@ -51,20 +51,16 @@ xtrabackup --prepare --apply-log-only --target-dir=/data/backups/mysql/ \
 
 ## Roll forward again to the second increment
 
-Roll forward again to the state on Wednesday:
+Roll forward again to the state on Wednesday (without --apply-log-only):
 
 ```
-xtrabackup --prepare --apply-log-only --target-dir=/data/backups/mysql/ \
+xtrabackup --prepare --target-dir=/data/backups/mysql/ \
    --incremental-dir=/data/backups/inc/wed/
 ```
 
-## Prepare the whole backup to be ready to use
-
-Create the new logs by preparing it:
+!!! note
 
-```
-xtrabackup --prepare --target-dir=/data/backups/mysql/
-```
+    Starting with [Percona Server for MySQL 8.0.30-22](release-notes/8.0/8.0.30-22.0.md), preparing a backup in two steps, --prepare --apply-log-only and then later --prepare, is not supported as dynamic metadata will be skipped causing corruption.
```

Comment:
Maybe?
and then a second `--prepare` is not supported. The second `--prepare` skips the dynamic metadata and causes corruption.

Reply:
Updated